icu4c/source/i18n/number_usageprefs.h

```diff
@@ -97,13 +97,13 @@ class U_I18N_API UnitConversionHandler : public MicroPropsGenerator, public UMem
     /**
      * Constructor.
      *
-     * @param inputUnit Specifies the input MeasureUnit. Mixed units are not
-     *     supported as input (because input is just a single decimal quantity).
-     * @param outputUnit Specifies the output MeasureUnit.
+     * @param inputUnit Specifies the input MeasureUnit. In case of Mixed unit, the input unit will be
+     * the biggest unit in the Mixed unit and the output will be the input unit. Otherwise, the input
+     * unit will be the output unit.
```

Comment:
Looks like it should rather be called "outputUnit" than "inputUnit"?

Reply:
I just want to convince that the specified input unit is the output unit ... we need to take this offline

Reply:
done offline

Reply:
"Otherwise, the input unit will be the output unit" - what is the output unit? This comment doesn't specify this at all. I'll fix.

Reply:
I've rewritten the sentences I found confusing. Marking as resolved.